tests/python/bls_test_template.py

```diff
@@ -119,7 +119,7 @@ def bls_particle_grid(N,
 
     max_num_particles_per_block = block_size**2 * 4096
 
-    x = ti.Vector(2, dt=ti.f32)
+    x = ti.Vector(2, dtype=ti.f32)
```

Comment:
```suggestion
    x = ti.Vector.field(2, dtype=ti.f32)
```
Please upgrade to the new API.

Btw, what backend do you have on your end?